src/index.ts

```diff
@@ -28,6 +28,7 @@ app.use(helmet());
  * Use the verifyToken to protect all the routes that require authentication
  */
 app.use("/example", verifyToken, exampleRoute);
+app.use("/inventory", require("./routes/inventoryRouter.ts"));
```

Comment:
Please move the require() into a seperate variable at the top of the file and call the method here. (examle: https://github.com/ChangePlusPlusVandy/ndc-backend/blob/main/src/routes/staffRouter.ts)